pwiz_tools/Skyline/SkylineGraphs.cs

```diff
@@ -1038,7 +1064,7 @@ void GraphSpectrum.IStateProvider.BuildSpectrumMenu(bool isProteomic, ZedGraphCo
             menuStrip.Items.Insert(iInsert++, toolStripSeparator14);
 
             // Need to test small mol
-            if (isProteomic)
+            if (isProteomic && control is GraphSpectrum)

```

Comment:
Too much use of "is GraphSpectrum". Better to use polymorphism, i.e. put something on your interface that both implementations can answer to distinguish behavior.

Reply:
Corrected.